examples/example.rules.kts

```diff
@@ -214,20 +214,18 @@ fun RuleSet.vulnerabilityInPackageRule() = packageRule("VULNERABILITY_IN_PACKAGE
 }
 
 fun RuleSet.highSeverityVulnerabilityInPackageRule() = packageRule("HIGH_SEVERITY_VULNERABILITY_IN_PACKAGE") {
-    val maxAcceptedSeverity = "5.0"
+    val thresholdScore = 5.0f
```

Comment:
Nit: `scoreThreshold` (it's a threshold for the score, not a score for the threshold)

Reply:
Though you could also read it as "threshold-score", so a score value that is used as a threshold. That would better match "threshold-severity" that I'm considering to introduce.